### ISSUE TITLE
Support custom ca for self signed certs

### ISSUE THREAD

**Issue by mrchief** [shoreditch-ops/artillery#199]
In addition to `-k` or   

```
tls: 
  rejectUnauthorized: false
```
options which completely disable ssl verify, it'd be nice to have an option to pass custom ca.  Something like:

```
tls: 
  ca: './ca-crt'
```

**Comment by hassy** [comment]
Anything set in `config.tls` will be merged into the Request options:

https://github.com/request/request#tlsssl-protocol

**Comment by mrchief** [comment]
Yes but that leaves the file reading bit to the user. Not a huge pain but now its not just a simple yaml/json config file.
